### PR TITLE
Update eslint from `v8` => `v9`

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-50941]

Also updated to `eslint-webpack-plugin`. Which needed to release a new version to be compatible with eslint v9.

### CODE REVIEW

**Comment by nmajor25** [comment]
Still looking into why the tests are borking. Not happening locally.

**Review by paulfalgout — CHANGES_REQUESTED** [review]
The borking tests are just the linter failing on this line: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/forms.js#L106

It's saying the `(e)` should be `()`

**Comment by nmajor25** [comment]
Yep, good call. Think we should be good now 🤞

File: src/js/services/forms.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default App.extend({
     try {
       store.set(this.getStoreId(), { submission, updated });
       this.trigger('update:submission', updated);
-    } catch (e) /* istanbul ignore next: Tested locally, test runner borks on CI */ {
```
Comment (paulfalgout) [inline src/js/services/forms.js:106]: the coverage drop is from the comment being removed

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/services/forms.js (hunk 1 above)
Comment (nmajor25) [reply src/js/services/forms.js:106]: Oops, added that back

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**